changelog/postblitQualifiers.dd

```diff
@@ -1,12 +1,13 @@
-Deprecate the use const/immutable/shared postblit
+Deprecate the use of a `const`/`immutable`/`shared` postblit
```

Comment:
I personally think nominal style would be better here, e.g. "Deprecation of  ..."